website/content/en/docs/main/tasks/private-networks/index.md

```diff
@@ -0,0 +1,394 @@
+---
+title: Private Networks
+description: Running ClusterLink in a private network when the K8s cluster is behind a NAT or firewall.
+---
+
+
+This task involves connecting ClusterLink behind a NAT or firewall.
+To connect the ClusterLink gateway, each peer should have a public IP that will be reachable from other peers to enable cross-cluster communications. In many cases, this is not possible because clusters are behind corporate NAT or firewalls that allow outgoing connections only. For this scenario, we will use the [Fast Reverse Proxy][] (FRP) open-source project to create reverse tunnels and connect all clusters behind a private network. With FRP, only one IP needs to be public to connect all the clusters in the fabric.
```

Comment:
The beginning of this sentence logically continues the previous sentence so it is confusing.
You probably mean that `Typically, each ClusterLink gateway has a public IP address, ... However, in some case, this is not possible...`